packages/vite-plugin-nitro/src/lib/vite-plugin-nitro.ts

```diff
@@ -104,9 +104,12 @@ export function nitro(options?: Options, nitroOptions?: NitroConfig): Plugin[] {
         }
 
         nitroConfig.alias = {
-          '#analog/ssr': normalizePath(
-            path.resolve(workspaceRoot, 'dist', rootDir, 'ssr/main.server')
-          ),
+          // This is not the final fix but start point to discuss a fix for windows
+          '#analog/ssr':
+            'file://' +
```

Comment:
You can use this to check for windows

```ts
import os from 'node:os'

const isWindows = os.platform() === 'win32'
```

```suggestion
            (isWindows ? 'file://' : '') +
```

Reply:
Thanks, i will add it and check tests in Windows after work